From bd20cdfb223ad117b041baee6e0a51cf5576c9ae Mon Sep 17 00:00:00 2001
From: Peter Geis <pgwipeout@gmail.com>
Date: Thu, 22 Jul 2021 14:37:00 -0400
Subject: [PATCH 094/478] arm64: dts: rockchip: enable sata1 on quartz64-a

Enable the sata1 port on the quartz64 model a.
This is muxed with the usb3 port, only one can be enabled at a time.

Signed-off-by: Peter Geis <pgwipeout@gmail.com>
---
 .../arm64/boot/dts/rockchip/rk3566-quartz64-a.dts | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts b/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
index c07e8cb9e971..798adafed7e5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
@@ -200,6 +200,11 @@ vcc_wl: vcc_wl {
 	};
 };
 
+&combphy1_usq {
+	status = "okay";
+	rockchip,enable-ssc;
+};
+
 &combphy2_psq {
 	status = "okay";
 };
@@ -569,6 +574,16 @@ vcc_sd_h: vcc-sd-h {
 	};
 };
 
+/* sata1 is muxed with the usb3 port */
+&sata1 {
+	status = "okay";
+};
+
+/* sata2 is muxed with the pcie2 slot*/
+&sata2 {
+	status = "disabled";
+};
+
 &sdhci {
 	bus-width = <8>;
 	mmc-hs200-1_8v;
-- 
2.35.3

